diff --git a/main.py b/main.py
index 239e013..26e7e34 100644
--- a/main.py
+++ b/main.py
@@ -5,16 +5,16 @@ from utils import seed_everything
 
 # Training settings
 hparams = {
-    "batch_size": 128,
-    "epochs": 20,
+    "batch_size": 512,
+    "epochs": 5,
     "lr": 3e-5,
     "gamma": 0.7,
     "image_size": 256,
-    "patch_size": 32,
-    "dim": 1024,
-    "depth": 24,
-    "heads": 16,
-    "mlp_dim": 4096,
+    "patch_size": 16,
+    "dim": 64,
+    "depth": 2,
+    "heads": 8,
+    "mlp_dim": 128,
     "frame_patch_size": 2,
     "temporal_depth": 6
 }
diff --git a/train.py b/train.py
index 99a6150..285a0cf 100644
--- a/train.py
+++ b/train.py
@@ -115,8 +115,8 @@ def train_vit(df, split_type, images_path, hparams):
         train_data = MAPFDataset(images_path, train_df, transform=train_transforms)
         val_data = MAPFDataset(images_path, test_df, transform=validation_transforms)
 
-        train_loader = DataLoader(dataset = train_data, batch_size=hparams["batch_size"], shuffle=True)
-        valid_loader = DataLoader(dataset = val_data, batch_size=hparams["batch_size"], shuffle=True)
+        train_loader = DataLoader(dataset = train_data, batch_size=hparams["batch_size"], shuffle=True, num_workers=8)
+        valid_loader = DataLoader(dataset = val_data, batch_size=hparams["batch_size"], shuffle=True, num_workers=8)
 
         model = SimpleViT(
             image_size = hparams["image_size"],
@@ -259,7 +259,7 @@ def validation_step(model, criterion, valid_loader, device):
         epoch_val_loss = 0
 
 
-        for data, labels, successes, runtimes in valid_loader:
+        for data, labels, successes, runtimes in tqdm(valid_loader, leave=False):
             data = data.to(device)
             labels = labels.to(device)
             successes = successes.to(device)
@@ -284,4 +284,4 @@ def validation_step(model, criterion, valid_loader, device):
         "eval/coverage": epoch_val_coverage,
         "eval/coverage runtime": epoch_val_coverage_runtime,
         "eval/loss": epoch_val_loss
-    }
\ No newline at end of file
+    }
